src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -374,13 +383,89 @@ impl DragonballInner {
         Ok((old_vcpus, new_vcpus))
     }
 
+    pub(crate) fn resize_memory(&mut self, req_mem_mb: u32) -> Result<(u32, MemoryConfig)> {
```

Comment:
we should add a sanity check here. If the req_mem > default_maxmemory, we should better ignore this resizing operation.

(Seems the toml_config in runtime-rs lacks this configuration, maybe we need to add this one first.)

Reply:
fixed